docs/content/en/blog/plugin-intro.md

```diff
@@ -0,0 +1,125 @@
+
+**Anyone can develop a Plugin freely.** We will pubilsh a guide on how to develop a Plugin in the future.
+
+Developing a Plugin will not be so tough. Here I introduce somes reasons:
+
+- The main task is implementing what you want to execute in each stage.
+  - Piped's core will control deployment flows and handle git. You don't need to deeply understand the mechanism of GitOps.
+  - To be exact, if you want Plan Preview or Drift Detection for the plugin, you need to implement them.
+
+- Plugins can be also written by languages other than Golang.
+
+- You don't need to put your plugin in PipeCD official repository.
+  - You can determine review policy and release timing as you want.
+
+## Advantages of Pluginnable PipeCD
+
+There are three big advantages. Thanks to them, **users will not need to manage additional CD tools** when adding a kind of platform or customizing behaviors.
+
+### 1. Users can deploy to platforms other than K8s/CloudRun/Terraform/ECS/Lambda
+
+Currently, PipeCD supports deploying to Kubernetes, CloudRun, Terraform, ECS, and Lambda.
+
+Plugins enable you to deploy to the other platforms, for example, Cloudflare, Azure, and Amazon CloudFormation/CDK.
+
+### 2. Users can choose behaviors as they want
+
+- Example 1: Deploy in a way different from built-in.
+- Example 2: For Kubernetes (or any platform), deploy simply for some Applications with Plugin-A, and deploy in a complicated way for the other Applications with Plugin-B.
+
+
+### 3. Users can develop and use custom Stages
+- Example 1: Analyze by New Relic or Amazon CloudWatch in the `ANALYSIS` stage.
+- Example 2: Run a Kubernetes Job or a Lambda function to execute a job.
+
+These effects will be more powerful as more Plugins come out since you can use plugins made by others.
+
+## Schedule overview
+
+![](/images/plugin-intro-schedule.drawio.png)
+
+Pluginnable PipeCD will be released around Feburary 2025.
+
+The Plugin development guide will be published after March 2025. After that, new Plugins will come out.
+
+For the current PipeCD users, please migrate to the Pluginnable version.  New features will be added to the Pluginnable version.
+We will share the details of migration ways later.  We've been making efforts to minimize the amount of the migration tasks. The main tasks will be modifying Piped configs and Application configs.
+
+## Conclusion
+
+Pluginnable PipeCD is a big step toward the PipeCD's vision - "**The One CD for All {applications, platforms, operations}**".  The community energy will be more important.
+
+To see the development progress, visit [the RFC](https://github.com/pipe-cd/pipecd/blob/master/docs/rfcs/0015-pipecd-plugin-arch-meta.md) or [the meta Issue](https://github.com/pipe-cd/pipecd/issues/5259).
+
+If you have any questions or opinions, feel free to talk in [CNCF Slack](https://cloud-native.slack.com/) > #pipecd-jp or [PipeCD Community Meeting](https://docs.google.com/document/d/1AtE0CQYbUV5wLfvAcl9mo9MyTCH52BuU7AngVUvE7vg/edit).
```

Comment:
[nits]
It's perhaps better to note on a non-Japanese channel.

```suggestion
If you have any questions or opinions, feel free to talk in [CNCF Slack](https://cloud-native.slack.com/) > #pipecd or [PipeCD Community Meeting](https://docs.google.com/document/d/1AtE0CQYbUV5wLfvAcl9mo9MyTCH52BuU7AngVUvE7vg/edit).
```

Reply:
@Warashi 
Thank you! I fixed in [7979946](https://github.com/pipe-cd/pipecd/pull/5372/commits/79799461b1836de20b763df756e62ccf3a5dfd2c)